app/view/main/Map.js

```diff
@@ -93,6 +93,21 @@ Ext.define('CpsiMapview.view.main.Map', {
      */
     addScaleBarToMap: true,
 
+    /*
+     * Flag that enables/disables permalink functionality
+     * @config {Boolean}
```

Comment:
This should be `@property` as well, shouldn't it?

Reply:
No, this is a config which can be passed from outside for initialization. A change of this option after the initialization would have no effect.